script.js

```diff
@@ -170,3 +171,23 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
   }
 });
+
+// STORE VALUES LOCALLY
+
+const inputText = document.querySelectorAll('.input-text');
+const dataStored = {
+  name: '',
+  email: '',
+};
```

Comment:
- [ ] it would be better to add the body text as well in order to complete all the requirements 
<img width="755" alt="Screenshot 2022-12-22 at 16 54 50" src="https://user-images.githubusercontent.com/61790587/209172909-2d194c2e-7ecb-4859-871c-354aa8b27e3b.png">